internal/pkg/manifest/initial_manifest_integration_test.go

```diff
@@ -1,3 +1,5 @@
+//go:build localintegration
```

Comment:
(I know this isn't related to your change)

Would you mind renaming the file from `initial_manifest_integration_test.go` to maybe `marshal_manifest_integration_test.go`?

(sidenote: It'd be dreamy if we can actually move all of our integ tests to a separate directory to make our source files a bit easier to parse and ensure we don't refer to internal state.)

Reply:
renamed for now, it would be a great idea to have all of our integ tests (in other packages too!) in a separate package from the areas they test! <img src="https://slackmojis.com/emojis/10804-meow_love/image/1643515051/meow_love.png" data-canonical-src="https://slackmojis.com/emojis/10804-meow_love/image/1643515051/meow_love.png" width="15" height="15" />